integration/lsp.test.ts

```diff
@@ -156,6 +156,6 @@ describe('module', () => {
         ast.body[0].init.value = '2';
         // Format into new source
         const src = format_from_js_file(ast);
-        expect(src).toBe('x = 2');
+        expect(src).toBe('x = 2\n');
```

Comment:
This all seems related to the formatter change. Just curious what the driving force behind this change was.

Reply:
I don't have the context for why it was necessary, but it was introduced here: https://github.com/influxdata/flux/pull/4905

Reply:
iirc the deal with this was the formatter behaved differently when run on the CLI vs through the LSP with regards to the trailing newline. This brought the two back into alignment.